jina/resources/k8s/template/deployment.yml

```diff
@@ -25,6 +25,13 @@ spec:
           - containerPort: {port_in}
           - containerPort: {port_out}
           - containerPort: {port_ctrl}
+        livenessProbe:
+          exec:
+            command:
+              - python
+              - from jina.peapods.runtimes.grpc.GRPCDataRuntime import GRPCDataRuntime; GRPCDataRuntime.is_ready(ctrl_address=0.0.0.0:port_ctrl)
+          initialDelaySeconds: 1
+          periodSeconds: 5
```

Comment:
Did we set the gRPC deadline? 5 seconds may be a little short IMO.

Reply:
This is just a first draft with some ideas and mostly pseudocode to share with the team. We have to figure out the good solution, maybe the best is to set it as part of the template or come with good defaults.

What would you suggest @gaocegege, your input is highly appreciated